.appveyor.yml

```diff
@@ -53,9 +53,8 @@ test_script:
 - cd c:\pillow
 - '%PYTHON%\%EXECUTABLE% -m pip install pytest pytest-cov pytest-timeout defusedxml ipython numpy olefile pyroma'
 - c:\"Program Files (x86)"\"Windows Kits"\10\Debuggers\x86\gflags.exe /p /enable %PYTHON%\%EXECUTABLE%
-- '%PYTHON%\%EXECUTABLE% -c "from PIL import Image"'
-- '%PYTHON%\%EXECUTABLE% -m pytest -vx --cov PIL --cov Tests --cov-report term --cov-report xml Tests'
-#- '%PYTHON%\%EXECUTABLE% test-installed.py -v -s %TEST_OPTIONS%' TODO TEST_OPTIONS with pytest?
+- PATH %PYTHON%;%PATH%
+- .ci\test.cmd
```

Comment:
Hmm. A minor concern perhaps, but your new version uses a hardcoded value for the Python executable, rather than the configuration from earlier in the file.

https://github.com/python-pillow/Pillow/blob/629c490020fbf3efe6766ae0bdca9690dc934fcc/.appveyor.yml#L17

Reply:
All of the test environments use a hardcoded executable name, and even in this file `%EXECUTABLE%` is always `python.exe`.